scripts/local-deploy.sh

```diff
@@ -17,6 +17,30 @@ current_git_url=$(git config --get remote.origin.url)
 current_git_branch=$(git rev-parse --abbrev-ref HEAD)
 
 machine_ip=$(hostname -I | cut -d ' ' -f1)
+devel_mode=0
+
+usage="$(basename "$0") [-h] [--key=value]
+
+Deploys all the osparc-ops stacks and the SIM-core stack on osparc.local.
+
+where keys are:
+    -h, --help  show this help text
+    --devel_mode             (default: ${devel_mode})"
+
+for i in "$@"
+do
+case $i in
+    --devel_mode=*)
+    devel_mode="${i#*=}"
+    shift # past argument=value
+    ;;
+    ##
+    :|*|--help|-h)
+    echo "$usage" >&2
+    exit 1
```

Comment:
MINOR: I saw this in an example. Always called my attention that ``myprogram --help`` would exit with failure command ..?